QUICK-START.md

```diff
@@ -5,7 +5,7 @@ Copyright © 2022 HMS Industrial Networks Inc.
 The Ewon Flexy Cumulocity Connector package provides a connector-based solution to Cumulocity for
 linking Ewon devices using a direct data path with a Flexy Java application.
 
-**Note**: This document provides a quick-start guide to use this connector, for a more comprehensive description of this process and the connector, please check out the [Installation and Configuration Guide](https://hmsnetworks.blob.core.windows.net/www/docs/librariesprovider10/downloads-monitored/software/flexy-cumulocity-integration-installation-and-configuration-guide-v1.0.pdf?sfvrsn=1490cbd7_5), the [Technical Documentation](https://hmsnetworks.blob.core.windows.net/www/docs/librariesprovider10/downloads-monitored/software/solution-pack---ewon-cumulocity-connector---technical-documentation.pdf), and the [Quick Start Demo Video](https://www.youtube.com/watch?v=LBy9LZPgM1U&ab_channel=CumulocityIoT).
+**Note**: This document provides a quick start guide to use this connector, for a more comprehensive description of this process and the connector, please check out the [Installation and Configuration Guide](https://hmsnetworks.blob.core.windows.net/www/docs/librariesprovider10/downloads-monitored/software/flexy-cumulocity-integration-installation-and-configuration-guide-v1.0.pdf?sfvrsn=1490cbd7_5), the [Technical Documentation](https://hmsnetworks.blob.core.windows.net/www/docs/librariesprovider10/downloads-monitored/software/solution-pack---ewon-cumulocity-connector---technical-documentation.pdf), and the [Quick Start Demo Video](https://www.youtube.com/watch?v=LBy9LZPgM1U&ab_channel=CumulocityIoT).
```

Comment:
This second commit can be merged with the original commit. (Leaving only one commit)

If you need help, let me know.